lib/runner.d.ts

```diff
@@ -0,0 +1,19 @@
+import { DatasetCore } from 'rdf-js'
+import { Writable } from 'stream'
+
+declare type Runner = {
+    (dataset: DatasetCore): Promise<any>
+}
+
+declare interface RunnerInit {
+    basePath: string
+    outputStream: Writable
+    pipeline?: string
+    variable?: Map<string, string>
+    verbose?: boolean
+    log?: (message: string) => void
+}
+
+declare function create (options: RunnerInit): Runner
+
+export = create
```

Comment:
Don't know what happened here, but doesn't look like it should be part of the PR.

Reply:
I did mention that in the PR description